testsuite/integration-arquillian/tests/base/src/test/resources/mapper-test/testrealm.json

```diff
@@ -0,0 +1,691 @@
+{
```

Comment:
Is it possible to remove this file from your PR and make sure that the Lightweight token test is based on the default file from `testsuite/integration-arquillian/tests/base/src/test/resources/testrealm.json` ? It looks that there are almost no changes between your file and the basic testsuite file. If any modifications are needed in the test, it is possible to either add them to the `LightWeightAccessTokenTest.addTestRealms` method (you already did some modifications here) or use admin client and update before/after particular test methods.

Sorry I missed this in previous reviews (if it was present). Having many various `testrealm-*.json` files means bigger work in the future when some testsuite maintenance is needed, so it is good to avoid adding more such files unless really needed.

Reply:
I removed mapper-test/testrealm.json and added it to addTestRealms.